generators/base-app/templates/webpack-config.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
```

Comment:
(double checked that no ovewrite was made on https://github.com/adobe/aio-lib-runtime/blob/a538deea9d307e95d0a8cda96bbb8fc5defc273a/src/build-actions.js#L92-L137)